donjo-app/libraries/Release.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+class Release
```

Comment:
Kalau class ini sengaja disusun untuk OpenSID (yaitu bukan library yg dikelola di tempat lain), maka perlu mengikuti aturan penulisan script OpenSID di https://github.com/OpenSID/OpenSID/wiki/Aturan-Penulisan-Script. Dan sekarang juga perlu ditambahkan pemberitahuan lisensi seperti yg sdh dipasang di file `assets/js/peta.js`.

Reply:
Resolved in d741f23